include/bluetooth/a2dp.h

```diff
@@ -16,6 +16,52 @@
 extern "C" {
 #endif
 
+/** @brief define the audio endpoint
+ *  @param name the endpoint variable name.
+ */
+#define BT_A2DP_ENDPOINT(name, role, codec, cap, config)\
```

Comment:
In LE Audio implementation we are going away from "cap" for capabilities as it is getting an overloaded term in Bluetooth Audio. I think @Vudentz ended on using "capability" instead. I suggest we streamline this across Zephyr

Reply:
Thanks, Has updated it.